server/src/main/kotlin/net/horizonsend/ion/server/miscellaneous/utils/Chat.kt

```diff
@@ -75,7 +75,7 @@ fun <T> Iterable<T>.joinToText(
 	return component
 }
 
-fun String.isAlphanumeric() = matches("^[a-zA-Z0-9]*$".toRegex())
+fun String.isAlphanumeric() = matches("^[a-z]*$".toRegex()) || matches("^[0-9]*$".toRegex()) || matches("^[a-z0-9]*$".toRegex())
```

Comment:
- isAlphanumeric is probably used elsewhere, so changing this could cause unintended bugs elsewhere.
- additionally, performing multiple regex checks is expensive, combine them into a single regex.